api/core/providers/anthropic/anthropic_provider.py

```diff
@@ -93,11 +105,12 @@ def _build_request(self, messages: list[MessageDeprecated], options: ProviderOpt
             ],
             model=options.model,
             temperature=options.temperature,
-            max_tokens=self._max_tokens(model_data, options.max_tokens),
+            max_tokens=self._max_tokens(model_data, options.max_tokens) + (thinking_budget or 0),
```

Comment:
that might push over the limit no ? Since I am guessing the thinking budget is counted in the max output tokens ?
Maybe integrate the thinking_budget in the `self._max_tokens` function to make sure `self.max_tokens` never returns a value > max output tokens ?

Reply:
Very well spotted. Thank you. Updated.https://github.com/WorkflowAI/WorkflowAI/pull/671/commits/1ddfdd0a639692a04a4435896c42e7be8bfec68b